pinax/stripe/views.py

```diff
@@ -195,15 +195,12 @@ class Webhook(View):
     def dispatch(self, *args, **kwargs):
         return super(Webhook, self).dispatch(*args, **kwargs)
 
-    def extract_json(self):
```

Comment:
I doubt anyone is subclassing `Webhook` and overriding `extract_json` but if they are this is a BIC.  I think we should just document this in the release notes (I'll take care of it).